extensions/binding/org.eclipse.smarthome.binding.onewire/src/main/java/org/eclipse/smarthome/binding/onewire/internal/handler/MultisensorThingHandler.java

```diff
@@ -50,8 +50,8 @@
  */
 @NonNullByDefault
 public class MultisensorThingHandler extends OwBaseThingHandler {
-    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<>(
-            Arrays.asList(THING_TYPE_MS_TH, THING_TYPE_MS_THS, THING_TYPE_MS_TV, THING_TYPE_AMS, THING_TYPE_BMS));
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<>(Arrays.asList(THING_TYPE_MS_TH,
+            THING_TYPE_MS_THS, THING_TYPE_MS_TV, THING_TYPE_AMS, THING_TYPE_BMS, THING_TYPE_TH));
 
```

Comment:
While this seems to work, in l. 90 a DS2438 is added. All thing handlers so far add the sensors that are really used. While the DS1923 and DS2438 look similar in the owfs, they will probably be different for other bridges (like a direct DS9490-support). You should add a new device for that. I'm not so sure how we can differntiate between the devices in the current thing handler. I'll have to think about that. My sugestions would be to move the DS1923 support to another PR so we can get the DS1420 support and your bug fixes merged as soon as possible.

Reply:
1. Not sure if I understand you correctly - do you want to create class for handling `DS1923` like you do for `DS18x23`? To be honest it was my first attempt to add support this way, but then I noticed that class `DS2438` is capable of handling various channels correctly. It is only matter of proper definition of the thing, like `ms-th` or `ms-tv`. Because of that I strongly recommend (as mentioned earlier) to rename `ms-th` into `ms-thv` (because it exposes 3 channels) and rename my `th` into `ms-th` as it really about temperature and humidity only.
1. Why do you bother about non OWFS support at this moment? OWFS provides good abstraction layer.
1. Do you want me to create one pull request for DS1420, one PR for issue with refresh and one more PR for issue with current channel?
1. What do you think about changing completely the definition of devices and do that in YAML file, something like this one? Of course it is not work for this moment, but something to consider. I think that the binding code could be simpler afterward.
```yaml
devices:
  - device: DS1923
    type: ms-th
    description: iButton Hygrochron Temperature/Humidity Logger with 8KB Datalog Memory
    channels:
      - channel: presence
        channel: humidity
        channel: temperature
          options: 
          - ignore85: [ true, false ]
  - device: DS18B20
    type: temperature
    description: Programmable Resolution 1-Wire Digital Thermometer 
    channels:
      - channel: presence
        channel: temperature
          options: 
          - ignore85: [ true, false ]
  - device: DS18B20
    type: temperature
    description: High-Precision 1-Wire Digital Thermometer 
    channels:
      - channel: presence
        channel: temperature
          options: 
          - resolution : [9, 10, 11]
            ignore85: [ true, false ]
```